### ISSUE TITLE
Add `fullName` to the session.profile object

### ISSUE THREAD

**Issue by crccheck** [CondeNast/launch-vehicle-fbm#33]
Right now, we provide a fullName for convenience of packaging first and last name. It'd be nice if we just stored that on `session.profile` too.

could be rolled into #32

**Comment by stripethree** [comment]
Currently the `profile` object is a direct copy of what comes back from the Graph API call for a user's public profile. I like this idea, but do have a small reservation about deviating from that paradigm... documentation can solve it though, I think.

**Comment by crccheck** [comment]
> is a direct copy of what comes back

Yeah, I noticed that when I tried using `firstName` vs `first_name`. We'll have to figure that out too. As for the original problem, I solved it by changing the template from `{fullName}` to `{first_name} {last_name}`
